### DISCUSSION TITLE
월드를 만들고 다른 사람이 제작한 아이템을 전시 또는 캐릭터에게 입힐 수 있는지 궁금합니다

### DISCUSSION THREAD

**Discussion by Yunkyung-Kim** [naverz/zepeto-studio-kor#447] [5. Q&A]
제 월드에서 제페토 상점에 파는 옷을 전시하거나 캐릭터에게 입힐 수 있나요?

**Comment by mayalhc** [comment]
공식 월드에서 적용 되고있으니까 월드에서도 가능은 할거 같습니다.

**Reply by x4cf** [reply]
현재는 불가능합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 입어보기 기능은 현재로서는 불가능하지만 추후에 가능하도록 준비 중에 있습니다 :)
다만 입어보기 기능에 다른 사람이 제작한 아이템은 지원이 어려울 수 있다는 점 참고 부탁 드립니다. 
감사합니다
